cookbooks/fb_apt/libraries/default.rb

```diff
@@ -43,5 +43,82 @@ def self._gen_apt_conf_entry(k, v, i = 0)
         return "\n#{indent}#{k} \"#{v}\";"
       end
     end
+
+    # Grab all keyrings owned by a package. We do not include
+    def self._get_owned_keyring_files
+      Dir.glob('/etc/apt/trusted.gpg.d/*').select do |keyring|
+        cmd = Mixlib::ShellOut.new("dpkg -S #{keyring}").run_command
```

Comment:
Always pass LANG=C when calling debian tools, otherwise you may get output in whatever the random locale the system is using